app/src/main/res/layout/activity_main.xml

```diff
@@ -6,13 +6,25 @@
     android:layout_height="match_parent"
     tools:context=".view.activity.MainActivity">
 
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
+    <com.google.android.material.bottomnavigation.BottomNavigationView
```

Comment:
The coloring of bottom navigation bar looks messed up. You can use `selector` for managing colors when an item is selected and not selected

Reply:
Yeah that happened after I converted it to androidx working on it right now